### ISSUE TITLE
Import tool?

### ISSUE THREAD

**Issue by willwade** [thedevs-network/kutt#37]
Has anyone created a tool maybe using the api to import bit.ly /a.n.other service links to Kutt.it (just before I embark on this!)

**Comment by poeti8** [comment]
This would be an interesting feature, since goo.gl is going to be down soon. Although, I should define a limit for the amount of links that a user can import. And there are also some possibilities that some of the links are already taken.

**Comment by willwade** [comment]
I’m not the best of coders so if someone else was to do this then fab. Yes - I really want to migrate from bitly but the goo.gl closure is a good use case too. It’s a good point about duplicates. Hmm. I would be hesitant on the limit aspect - but I understand the concerns.

**Comment by trgwii** [comment]
General notes:

* Check what types of export filetypes other services are using, pick the most widely supported 1-2 formats.

* On limits: Why not have a general per-user limit of total links instead of per-import? (Say 10k per user?)

**Comment by willwade** [comment]
Bit.ly. Only uses API. Uses oAuth 2.0. Full history via [/v3/user/link_history](http://dev.bitly.com/user_info.html#v3_user_link_history) (Interestingly they have a [guide transitioning from Google](http://dev.bitly.com/transition_from_google.html))

**Comment by skanderbouricha1** [comment]
I agree that having import/export options would make migrating from services like Bitly or goo.gl much easier. About the limits - maybe a tiered approach could work? For example, free users get 1k links, while paid/pro users get 10k or more.

Also, +1 for supporting common file formats like CSV or JSON for exports.
